bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/gemini/gmos/GmosNorthType.java

```diff
@@ -171,8 +171,8 @@ public DisperserNorth parse(String name, DisperserNorth defaultValue) {
         OIIIC_G0319("OIIIC_G0319", "OIIIC", "0.514"),
         HeII_G0320("HeII_G0320", "HeII", "0.468"),
         HeIIC_G0321("HeIIC_G0321", "HeIIC", "0.478"),
-        OVI_G0345_20000053("OVI_G0345", "OVI", "0.6835"),
-        OVIC_G0346_20000054("OVIC_G0346", "OVIC", "0.678"),
+        OVI_G0345("OVI_G0345", "OVI", "0.6835"),
```

Comment:
Could this cause troubles with existing programs? Do we have these filters on the current production ODB?

Reply:
I recently added them to the 2018A OCS and they are new, so I am thinking this should be fine.

Reply:
👍